src/source/avro.rs

```diff
@@ -82,6 +83,22 @@ impl From<Vec<u8>> for Payload {
         let mut cursor = Cursor::new(vec);
         let header: Header = (&mut cursor).into();
 
+        let mut key_pairs = HashMap::new();
+        // Read key pairs from wire if version > 1
```

Comment:
We will need to update the wiki documentation after this is merged.  See - https://github.com/postmates/cernan/wiki/SourcesAvro